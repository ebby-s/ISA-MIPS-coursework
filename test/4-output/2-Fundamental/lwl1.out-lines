TB : INFO : register_v0=01000000
TB : finished; active=0
