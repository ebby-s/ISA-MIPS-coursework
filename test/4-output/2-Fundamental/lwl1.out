01000000
0
